apps/balancer-tools/src/utils/formatDate.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Formats a date in the "MM/DD/YYYY" (American) format.
+ *
+ * @param {Date} date - The input date to be formatted.
+ * @returns {string} A string representing the formatted date.
+ */
+export function formatDateToAmerican(date: Date): string {
```

Comment:
```suggestion
export function formatDate(date: Date): string {
```

`#would` also move this to the `utils` package and add a test to it